packages/fxa-auth-server/test/local/senders/mjml-emails.ts

```diff
@@ -0,0 +1,316 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* TODO: test translations with Fluent
+ * NOTE: This file tests `email.js` and the MJML/EJS templates in `lib/senders/emails`.
+ * Eventually we will retire `email.js` but as templates are being converted over to
+ * the new stack, tests per template will be added, and then any other mailer tests
+ * will be copied over.
+ */
+
+const ROOT_DIR = '../../..';
+
+import { assert } from 'chai';
+import mocks from '../../mocks';
+import proxyquire from 'proxyquire';
+import { URL } from 'url';
+
+const config = require(`${ROOT_DIR}/config`).getProperties();
+if (!config.smtp.sesConfigurationSet) {
+  config.smtp.sesConfigurationSet = 'ses-config';
+}
+
+const TEMPLATE_VERSIONS = require(`${ROOT_DIR}/lib/senders/templates/_versions.json`);
+
+const MESSAGE = {
+  // Note: acceptLanguage is not just a single locale
+  acceptLanguage: 'en;q=0.8,en-US;q=0.5,en;q=0.3"',
+  appStoreLink: 'https://example.com/app-store',
+  code: 'abc123',
+  date: 'Wednesday, Apr 7, 2021',
+  deviceId: 'foo',
+  location: {
+    city: 'Mountain View',
+    country: 'USA',
+    stateCode: 'CA',
+  },
+  email: 'b@c.com',
+  flowBeginTime: Date.now(),
+  flowId: '0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef',
+  ip: '219.129.234.194',
+  locations: [],
+  numberRemaining: 2,
+  primaryEmail: 'c@d.com',
+  service: 'sync',
+  time: '5:48:20 PM (PDT)',
+  timeZone: 'America/Los_Angeles',
+  tokenCode: 'abc123',
+  type: 'secondary',
+  uaBrowser: 'Firefox',
+  uaBrowserVersion: '70.0a1',
+  uaOS: 'Windows',
+  uaOSVersion: '10',
+  uid: 'uid',
+  unblockCode: 'AS6334PK',
+};
+
+// key = query param name, value = MESSAGE property name
+const MESSAGE_PARAMS = new Map([
+  ['code', 'code'],
+  ['email', 'email'],
+  ['primary_email_verified', 'email'],
+  ['product_id', 'productId'],
+  ['plan_id', 'planId'],
+  ['secondary_email_verified', 'email'],
+  ['service', 'service'],
+  ['uid', 'uid'],
+  ['unblockCode', 'unblockCode'],
+]);
+
+interface Test {
+  test: 'equal' | 'include' | 'notInclude' | any;
+  expected: string;
+}
+
+// prettier-ignore
+const COMMON_TESTS = new Map<string, Test | any>([
```

Comment:
Nit: It looks like this block has 3 space indentations, can we make it 2 to be consistent?